c3po/api/service/feed_service.py

```diff
@@ -194,3 +197,39 @@ def get_frequent_posts(url, start, limit):
                     "message": "Try again",
                 }
                 return response_object, 500
+
+    @staticmethod
+    def get_underrated_posts(url, start, limit):
+        with session_scope() as session:
+            try:
+                total = session.query(UserPosts).count()
+                posts = (
+                    session.query(UserPosts)
+                    .order_by(UserPosts.share_date.desc())
```

Comment:
Can you move this to be after the joins? I suspect it won't make a difference but usually, a query is structured as `SELECT - FROM( - JOIN) - WHERE - ORDER BY - LIMIT`.

Reply:
You can have multiple `ORDER BY` queries and they will be applied in the order in which they appear in the query.

Reply:
@ghostwriternr according to [this](https://stackoverflow.com/questions/15791760/how-can-i-do-multiple-order-by-in-flask-sqlalchemy) we can do order_by on multiple columns. Shall I do this instead? Seems like the thing we want to implement

Reply:
Ok it is basically the same thing :v: I'll make the changes